WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -0,0 +1,258 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
+    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, that implements trustless CoinJoin over the Tor anonymity network.">
+    <meta http-equiv="onion-location" content="http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion" />
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://wasabiwallet.io">
+    <meta property="og:title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
+    <meta property="og:description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, it implements trustless CoinJoin over the Tor anonymity network.">
+    <meta property="og:image" content="https://wasabiwallet.io/images/wasabi-social.jpg">
+    <meta property="og:site_name" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image">
+    <meta property="twitter:url" content="https://wasabiwallet.io">
```

Comment:
Is this correct? Shouldn't it be the url to the twitter account?

Reply:
Yes, its correct. It's for sharing the website on twitter.